doc/sphinx/Clusters_from_Scratch/active-passive.rst

```diff
@@ -67,81 +67,74 @@ a specific OCF provider (the **IPaddr2** part of **ocf:heartbeat:IPaddr2**), run
 
     [root@pcmk-1 ~]# pcs resource agents ocf:heartbeat
     apache
-    aws-vpc-move-ip
-    aws-vpc-route53
-    awseip
-    awsvip
-    azure-events
+    conntrackd
+    corosync-qnetd
     .
     . (skipping lots of resources to save space)
     .
-    symlink
-    tomcat
-    vdo-vol
     VirtualDomain
     Xinetd
 
+If you want to list all resource agents available on the system, run **pcs
+resource list**. We'll skip that here.
+
 Now, verify that the IP resource has been added, and display the cluster's
-status to see that it is now active:
+status to see that it is now active. Note: There should be a stonith device by
+now, but it's okay if it doesn't look like the one below.
 
 .. code-block:: none
 
     [root@pcmk-1 ~]# pcs status
     Cluster name: mycluster
     Cluster Summary:
       * Stack: corosync
-      * Current DC: pcmk-2 (version 2.0.5-4.el8-ba59be7122) - partition with quorum
-      * Last updated: Tue Jan 26 19:22:10 2021
-      * Last change:  Tue Jan 26 19:20:28 2021 by root via cibadmin on pcmk-1
+      * Current DC: pcmk-1 (version 2.1.2-4.el9-ada5c3b36e2) - partition with quorum
+      * Last updated: Wed Jul 27 00:37:28 2022
+      * Last change:  Wed Jul 27 00:37:14 2022 by root via cibadmin on pcmk-1
       * 2 nodes configured
-      * 1 resource instance configured
-    
+      * 2 resource instances configured
+
     Node List:
       * Online: [ pcmk-1 pcmk-2 ]
 
     Full List of Resources:
-      * ClusterIP	(ocf::heartbeat:IPaddr2):	 Started pcmk-1
+      * xvm	(stonith:fence_xvm):	 Started pcmk-1
```

Comment:
I think we should name the fencing resource something generic, then we just have to manually replace "fence_xvm" here with what's used in the example.

I've got mixed feelings about what to use in the example. I would think people are more likely to use VMs these days for the walk-through, but I'd rather not get into the weeds of VM configuration (including firewall, fence-virt, etc.). I think keeping fence_ipmilan for the example might be better, but then add a fence-virt walk-through either in an appendix or on the wiki (with a mention here with a link).

Reply:
> I think we should name the fencing resource something generic

That's simple enough, something like `fence_dev` and maybe `stonith:some_fence_agent` or `stonith:some_agent`.

> I'd rather not get into the weeds of VM configuration

100%. I've been dreading this, though it's totally doable.